pykern/pkcli/__init__.py

```diff
@@ -162,6 +171,24 @@ def main(root_pkg, argv=None):
     return 0
 
 
+def _argh_name_mapping_policy():
+    """Set the argh name_mapping_policy.
```

Comment:
`Return name_mapping_policy based on feature test of NameMappingPolicy`

Reply:
7ff4012